### PR TITLE
Tag TimerOutputs.jl v0.2.0

### PR DESCRIPTION
**Pull request by attobot**
Repository: [KristofferC/TimerOutputs.jl](https://github.com/KristofferC/TimerOutputs.jl)
Release: [v0.2.0](https://github.com/KristofferC/TimerOutputs.jl/releases/tag/v0.2.0)
Travis: [Travis Build Status](https://travis-ci.org/KristofferC/TimerOutputs.jl/builds/207382977)
Diff: [vs v0.0.1](https://github.com/KristofferC/TimerOutputs.jl/compare/64d0501da0de4bba62b7d60b9b67122ec62f3e51...9a548622a5e7907d01618fb7ce4d85cffbe73c4e)
`requires` vs v0.0.1: 
```diff
--- 0.0.1/requires
+++ 0.2.0/requires
@@ -1 +1,3 @@
-julia 0.4
+julia 0.5
+Compat 0.17
+Crayons
```
cc: @KristofferC

### CODE REVIEW

File: TimerOutputs/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+Compat 0.17
```
Comment (tkelman) [inline TimerOutputs/versions/0.2.0/requires]: you're not using any new style abstract types. are you using compat at all right now?

Comment (KristofferC) [inline TimerOutputs/versions/0.2.0/requires]: I cleaned up the code since last time and apparently I don't need compat anymore!

**Comment by KristofferC** [comment]
No more compat

**Comment by KristofferC** [comment]
Don't merge. Forgot some stuff (like precompilation)

**Pull request merged**